### DISCUSSION TITLE
How can I synchronize the timing of the two UE?

### DISCUSSION THREAD

**Discussion by Kong-TaeHyeon** [srsran/srsRAN_Project#936] [General Help] [answered]
I would like to create a situation where the two srsUEs have the same preamble, and at the same time, ask the enb for rach to create a situation where the collision occurs.

How can we match the timing of the two srsUEs?

**Comment by pgawlowicz** [comment, marked as answer]
If you run the setup over ZMQ (with a gnuradio broker), and set the number of preambles to 1, you will get a collision. 

It is more challenging to achieve it over the air. You can try to first start two srsUEs and then start gnb. There is a chance, they will probably sync to the cell and send a PRACH simultaneously.

**Reply by Kong-TaeHyeon** [reply]
Thank you sir! 

Can I ask other Questions . ..  ?? 
1. https://github.com/srsran/srsRAN_4G/discussions/1420
2. I run over zeroMQ but After trying to connect once and terminating again (crtl + c), when uE is activated, the eNB is not found So I have to turn on eNB again and turn on uE. . !
